river: Add various string functions to stdlib (#4388)
grafana/agent: docs/sources/flow/reference/stdlib/replace.md

---
aliases:
- ../../configuration-language/standard-library/replace/
title: replace
---

# replace

> `replace` searches a given string for another given substring, and replaces each occurrence with a given replacement string.

Reviewer: ```suggestion
`replace` searches a string for a substring, and replaces each occurrence of the substring with a replacement string.
```

Could we simplify this and remove all the givens?
